### DISCUSSION TITLE
Feature request: persist config / calibration

### DISCUSSION THREAD

**Discussion by jpettitt** [ESPresense/ESPresense-companion#373] [Q&A]
It seems like the config / calibration is persisted in mqtt messages, the issue is those go away if I reboot Mosquitto.  I'd like them to be persisted someplace where they survive a hard reboot for the server.  I've mitigated it a little by moving the MQTT server out of HA into it's own container on proxmox which at least means I don't lose the config on every HA operating system update ...

**Comment by DTTerastar** [comment]
Make sure you're not setup to ignore the retain option.  Retain should save it in mqtt and survive reboots.
